Logout function

DelaMarc opened the pull request:
Added a logout function that clears the user cookies and redirects the user to the login page.
Also made a navigation bar template and included it on the according website pages. Thus, if the website's navbar has to be modified, we'll have only one file to change.

DelaMarc committed:
Merge pull request #1 from YashasG98/master

syncing

DelaMarc committed:
redirect user if they try to access pages without being logged in

DelaMarc committed:
added error message on login.html

DelaMarc committed:
Added a logout function that clears the user cookies and redirects the user to the login page.
Also made a navigation bar template and included it on the according website pages.

YashasG98 commented on server.py:
```diff
@@ -103,173 +114,160 @@ def Login():
     return render_template('login.html', error = error)
 
-@app.route('/profile', methods = ['GET','POST'])
+@app.route('/logout', methods = ['GET', 'POST'])
```
Why exactly is GET required here?

DelaMarc replied:
For nothing. My bad, it's removed now.

YashasG98 commented on templates/index.html:
```diff
@@ -39,7 +39,7 @@
+    </nav>-->
     <div style='height: 100vh;' class="jumbotron">
         <h1 class="display-4">Hello, gamer!</h1>
         <p class="lead">Welcome to the abode of multiplayer gaming!</p>
```
Please add the logout functionality to the exit button on this page. Also, while you're at it, rename it to 'Log out'

DelaMarc replied:
I pushed the branch

YashasG98 reviewed (changes requested):
This is amazing @DelaMarc ! Please make the minor changes requested and add the sessionNavigationBar template

DelaMarc committed:
Removed the requirements for the logout function in server.py
Removed commented navbar and modified 'Exit' button to 'Log out' button in index.html
Changed the target of the Login button in login.html
